arch/ext/S.yaml

```
@@ -333,3 +333,33 @@ params:
       When not implemented mstatus.TVM will be read-only-zero.
     schema:
       type: boolean
+  MSTATEEN_ENVCFG_TYPE:
+    when:
+      name: Smstateen
+      version: ~> 1.0
```

Missed the dash before "name" here:
```suggestion
    when:
￼     - name: Smstateen
￼       version: ~> 1.0
```